routing/prefix_tree/node.go

```diff
@@ -0,0 +1,114 @@
+// Copyright (c) 2016 Bitfury Group Limited
+// Distributed under the MIT software license, see the accompanying
+// file LICENSE or http://www.opensource.org/licenses/mit-license.php
+
+package prefix_tree
+
+import "errors"
+
+type Node interface {
```

Comment:
It is useful when creating an image of the routing table. Moved into routing/rt/visualisation .